### PR TITLE
Add Node.js 0.12.x and 0.10.x versions for test

### PR DESCRIPTION
**Pull request by madarche**
And BTW do you consider this module to still be relevant and up-to-date with current stable Node.js? I am asking because this repository hasn't changed since 2 years. But it looks fine to me.

Thanks

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,9 +1,11 @@
 language: node_js
 node_js:
+  - 0.12
+  - 0.10
   - 0.8
```
Comment (rvagg) [inline .travis.yml]: can you remove 0.8 while you're in here please?

**Comment by madarche** [comment]
@rvagg : done.

And what about the relevancy of this module?

**Comment by madarche** [comment]
@rvagg : your last commits and 1.0.0 version indicate the relevancy of the module question very well :-) Thanks :-) Could you then accept this PR?

**Comment by rvagg** [comment]
done, so regarding relevancy .. I don't use this module but it's in pretty high use:

who?

I'd opt for [bl](https://github.com/rvagg/bl) for most use-cases here, it also has the benefit of being streams2 while this is still streams1. But if it's working for people then I'm not going to judge!

**Comment by madarche** [comment]
Thanks a lot @rvagg for the detailed answer!

**Pull request merged**